tests/buffer/buffer_api_common.h

```diff
@@ -362,7 +364,11 @@ void test_buffer(util::logger &log, cl::sycl::range<dims> &r,
       sub_r[0] = r[0] - i[0];
       cl::sycl::buffer<T, dims> buf_sub(buf, i, sub_r);
       auto isSubBuffer = buf_sub.is_sub_buffer();
+      auto notSubBuffer = buf.is_sub_buffer();
```

Comment:
The two names seem a bit strange. Maybe:
```suggestion
      auto isOrigSubBuffer = buf.is_sub_buffer();
```

But I'm not sure it's even allowed in the spec to construct a sub-buffer from a sub-buffer.

Reply:
Done.